src/lib/colors.js

```diff
@@ -0,0 +1,150 @@
+export const colors = [
```

Comment:
Nice idea with these colors. However, I feel the majority of the colors in this array would not be used or required and some individuals might just call in the colors they need than referring to this file to pick a color, @chikeozulumba what do you think? any clarifications?

Reply:
@chikeozulumba you might also want to reduce the file changes...good job!

Reply:
> @chikeozulumba you might also want to reduce the file changes...good job!

I removed a lot of unnecessary files hence the files changes.